lib/rules/no-extra-boolean-cast.js

```diff
@@ -17,6 +17,8 @@ const astUtils = require("../util/ast-utils");
 
 module.exports = {
     meta: {
+        type: "problem",
```

Comment:
I'd consider `no-extra-boolean-cast` to be a `suggestion` -- it seems like it improves the style of code but probably wouldn't be pointing out a bug.

Reply:
I agree. Let's see what others think.

Reply:
Agreed, this should be a "suggestion".

Reply:
Agreed.

Reply:
Set to "suggestion"